test/puppetlabs/trapperkeeper/services/authorization/authorization_service_test.clj

```diff
@@ -89,9 +137,22 @@
             {:keys [status body]} (app req)]
         (is (= status 200))
         (is (= body "Prefix: !dlroW olleH")))))
+  (testing "(TK-260) Authorizing unauthenticated requests"
+    (let [app (build-ring-handler default-rules)]
+      (let [{:keys [status body]} (app unauthenticated-request)]
+        (is (= status 403))
+        (is (= body "global deny all - no rules matched")))
+      (let [req (assoc unauthenticated-request :uri "/puppet-ca/v1/certificate/ca"
+                                               :body "FOOBAR")
+            {:keys [status body]} (app req)]
+        (is (= status 200))
+        (is (= body "Prefix: RABOOF")))
+      (let [req (assoc unauthenticated-request :uri "/not/covered/by/rules")
+            {:keys [status body]} (app req)]
+        (is (= status 403)))))
```

Comment:
Seems like here we should either have an extra assertion for the value of `body` or just remove the `body` key from what's being destructured since, in the latter case, nothing would be using it.